Adyen/Model/PaymentComponentData.swift

```diff
@@ -87,6 +89,10 @@ public struct PaymentComponentData {
         return paymentMethod.delegatedAuthenticationData
     }
     
+    /// An encoded string containing important SDK-specific data.
+    /// It is recommended to pass this field to your server to ensure maximum performance and reliability.
```

Comment:
Please disregard if it makes no sense, but I decided to bring this to your attention. Can we make this statement less vague and explain a bit more?
And the second part, a bit more bold just to stretch us - can we make it non-optional?